linux/kernel/building.md

```diff
@@ -64,7 +64,9 @@ KERNEL=kernel7l
 make bcm2711_defconfig
 ```
 
-After this is done, you will likely want to adjust the `LOCALVERSION` to ensure your new kernel does not get the same version string as the upstream kernel. This both clarifies you are running your own kernel in the output of `uname` and ensures existing modules in `/lib/modules` are not overwritten.
+#### Customising the Kernel version using LOCALVERSION
```

Comment:
This would make more sense with one fewer `#`, since it doesn't belong to the list formed by the items above.

Reply:
But OTOH it probably *does* belong under the `### Kernel configuration` section? :man_shrugging: 
Maybe we need a new `#### Apply the default configuration` header, and then move the "Raspberry Pi 4" etc. headers another level down? How many levels of subheadings does https://www.raspberrypi.org/documentation/ support?

Reply:
(Editing comments doesn't trigger notifications...)
That looks OK to me.